deploy/charts/harvester/dependency_charts/whereabouts/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: whereabouts
```

Comment:
probably a name with `whereabouts-cni` is more preferred. Additionally, can we please add a README page for this chart to describe which repo it was referring to and why we need this dependency-chart, thanks.

Reply:
i will consider to keep it as upstream.
So I will add readme for it.